mas 2.2.1

rgoldberg opened the pull request:
Created by `brew bump`

---

Created with `brew bump-formula-pr`.

rgoldberg commented:
@bevanjkay @chenrui333 Must I change the commit message again? I guess I might need to prefix the message with `mas: `. I thought that was only for PR titles, not for every commit. I don't want to break your procedure, but I also don't want to cause all the checks to restart again unnecessarily if changing the commit message isn't necessary. Thanks.

bevanjkay commented:
Any additional changes required for a version bump are usually squashed into a single commit, in this case with the message `mas 2.2.1` - feel free to force push again to fix it, the alternative is to leave it and @BrewTestBot will open another PR with the commits squashed (cherry-picked to attribute them to you), but this runs some CI again anyway so won't necessarily save time.

rgoldberg commented:
@bevanjkay Just to ensure that I correctly understand you, the whole commit message should be `mas 2.2.1`; it shouldn't include any information about the purpose of the commit, e.g., it shouldn't be `mas 2.2.1: Upgrade Xcode to 15.0 to support Swift 5.9.`.

If I'm incorrect, can you give me a full commit message that should work, include all lines if it's multiline?

I just don't want to kick off a whole other set of checks if the commit message check will fail again.

Thanks.

rgoldberg commented:
@bevanjkay I understand now. I thought that you mean that the additional changes are usually squashed into a single commit by your `autosquash` tag. But I realized that if I squashed it all beforehand, the commit should pass the commit message check, which it did.

bevanjkay commented:
Thanks for maintaining `mas` by the way! Happy user over here, especially with the recent revamp 👏

rgoldberg commented:
@bevanjkay Thanks for working on Homebrew!

I appreciate all the new features & performance improvements, plus the general maintenance work for the constantly updating formulae & casks that you guys need to do.

I don't have the same maintenance burden in mas, but I do have Apple deciding to break private frameworks, as with macOS 15.4 breaking the installed app list…

I have a host of improvements for mas in the pipeline. Recently, I fixed the 15.4 catastrophe, a few wrinkles from the shift to Spotlight & to Swift Concurrency, and I've cleaned up a ton of the code in preparation for larger scale changes ahead (like json output, continually updated completion scripts for zsh, bash & fish, supporting iOS & IPadOS apps, supporting bundle IDs in addition to app IDs, accepting multiple IDs in a single command for all commands, fixing `signin` & `account`, fixing multiple issues with versions for `outdated` & `upgrade`, command aliases, etc.).
